test/app_maven.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var mvnpath;
+
+exports.setMavenPath = function (path) {
+    mvnpath = path;
+}
+
+function execCamelRun() {
+    return new Promise((resolve, reject) => {
+        let alldata = '';
+        console.log('trying to run mvn camel:run in ' + mvnpath);
+        try {
+            const spawn = require('child_process').spawn;
+            const child = spawn("mvn", [ "camel:run" ], { cwd: mvnpath });
+            child.on('stdout', function (data) {
+                alldata += data;
+                if (alldata.includes('INFO: Creating Service')) {
+                    console.log('Service created');
+                    resolve();
+                };
+            });
+            child.on('error', function (err) {
+                console.log('Error encountered: ' + err);
+                reject(err);
+            });
+        } catch (e) {
+            console.error(e);
+            reject();
+        }
+    });
+}
+
+function execCamelRun2(mvn) {
+    return new Promise((resolve, reject) => {
+        console.log('Running mvn camel:run');
+        mvn.execute(['camel:run']).then(() => resolve()).catch((e) => reject(e));
+    });
+}
+
+exports.doCamelRun = function () {
+    return new Promise((resolve, reject) => {
+        const mvn = require('maven').create({
+            cwd: mvnpath
+        });
+        mvn.execute(['clean', 'install']).then(() => {
+            console.log('Running');
+            // Resolve or reject based on the result of the camel run.
+            execCamelRun().then(() => resolve()).catch((e) => reject(e));
+        }).catch((e) => reject(e));
+        // console.log('Running mvn install');
```

Comment:
commented code to remove

Reply:
Removed file, which isn't needed any longer when I removed the skipped test